### PR TITLE
Aws

### PR DESCRIPTION
**Pull request by c-ryan747**
Added basic AWS integration

### CODE REVIEW

File: src/main/java/uk/ac/cam/cl/quebec/face/CredentialsManager.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+     * [default]
+     * aws_access_key_id = YOUR_ACCESS_KEY_ID
+     * aws_secret_access_key = YOUR_SECRET_ACCESS_KEY
+     */
+
+    private CredentialsManager() {}
+
+    static AWSCredentials getCredentials() {
+        if (credentials == null) {
+            credentials = null;
```
Comment (PML369) [inline src/main/java/uk/ac/cam/cl/quebec/face/CredentialsManager.java]: This looks very auto-generated, but... know what this line is doing here?

File: src/main/java/uk/ac/cam/cl/quebec/face/S3AssetDownloader.java [hunk 2]
```diff
@@ -1,28 +1,77 @@
...
+        s3.setRegion(region);
+
+        File file = null;
+
+        try {
+            String bucketName = "quebec-userfiles-mobilehub-1062763500";
+            S3Object object = s3.getObject(new GetObjectRequest(bucketName, url));
+            // handle object.getObjectContent() somehow
+            file = File.createTempFile("", "url");
+            file.deleteOnExit();
```
Comment (PML369) [inline src/main/java/uk/ac/cam/cl/quebec/face/S3AssetDownloader.java]: If this is here, do we need the list of temp files?

Comment (c-ryan747) [reply src/main/java/uk/ac/cam/cl/quebec/face/S3AssetDownloader.java]: Probably not, should probably delete when you're done with the file anyway

File: src/main/java/uk/ac/cam/cl/quebec/face/CredentialsManager.java (hunk 1 above)
Comment (c-ryan747) [reply src/main/java/uk/ac/cam/cl/quebec/face/CredentialsManager.java]: Ooops

**Pull request closed without merging**